team_structure.md

```diff
@@ -0,0 +1,95 @@
+# Team Structure
+
+## Org Chart & Interfaces
+
+```mermaid
+flowchart TD
+    CEO{CEO};
+    CTO{CTO};
+    PO{Product Owner};
+    team{voiio};
+    TM{tech. Manager};
+    TL{dep. Head};
+    CEO -- compnay OKR --> CTO;
+    CTO -- product OKR --> PO;
+    CTO <-. platform architecture .-> TM;
+    PO -- project OKR --> TM;
+    TM <-- project collab. --> TL;
+    PO <-- roadmap planing --> TL;
+    PO -- product update --> team;
+```
+
+## Artifacts & Ownership
+
+```mermaid
+flowchart LR
+    CEO{CEO};
+    CTO{CTO};
+    PO{PO};
+    TM{TM};
+    companyVision[[Company Vision]];
+    productVision[[Product Vision]];
+    productRoadmap[[Product Roadmap]];
+    companyObjectives[[Company Objectives]];
+    companyKeyResults[[Company Key Results]];
+    productObjectives[[Product Objectives]];
+    productKeyResults[[Product Key Results]];
+    projectKeyResults[[Project Key Results]];
+    projectObjectives[[Project Objectives]];
+    project[[Project]];
+    CEO -. owns .- companyVision;
+    CEO -. sets .- companyObjectives;
+    CTO -. delivers .- companyKeyResults;
+    CTO -. owns .- productVision;
+    CTO -. set .- productObjectives;
+    PO -. delivers .- productKeyResults;
+    PO -. owns .- productRoadmap;
+    PO -. sets .- projectObjectives;
+    TM -. delivers .- projectKeyResults;
+    TM -. owns .- project;
+```
+
+### Visions
+
+A clear vision needs to be set for the company and all departments. Those documents
+need to be shared with all employees.
+
+### OKR
+
+_ORK = Objective Key Result_
+
+We use OKRs as a bidirectional communication, with goal alignment in the form of
+objectives and impactful delivery in form of key results. They must be agreed upon
+by both the team and manager.
+
+ORKs are communicated within the company to identify synergies or conflicts of interest.
+
+#### Objectives
+
+Objectives are set by the manager in alignment with the company or product vision.
+They are their main tool of control and need to be highly specific and achievable.
+Objectives may change over time, to allow the manager to set the focus on new challenges.
+
+#### Key Results
+
+Key results are committed output on the objective set by the manager.
+They must be achievable by the team without external dependencies or side effects.
+Should a team struggle to define key results, this might indicate that the manager
+needs to set a more refined objective.
+
+### Product Roadmap
```

Comment:
We need to explain a bit more detailed how department heads can get projects on our roadmap (it is also not fully clear to me). But I guess the process should be that project requests from other departments should already be aligned with the company vision/objective, because otherwise I will have to say no to a lot of requests. 
Overall it would make sense if every department could follow a similar OKR structure. If not, I could prioritise project ideas individually and challenge our own product objectives in order to get something completely different on the roadmap, but that should be an exception. It should be clear that since product objectives are coming from you and (indirectly) from Björn, I am quite restricted in planning and we're obviously also a super small team with limited capacities. So department heads need to make sure that their project ideas are already aligned. I am emphasising that, because looking on the structure it could appear that department heads can put whatever they want on roadmap as long as they're talking to me and I'll make a project objective out of it.